fpmsyncd/fpmlink.cpp

```diff
@@ -120,12 +210,35 @@ uint64_t FpmLink::readData()
 
         if (hdr->msg_type == FPM_MSG_TYPE_NETLINK)
         {
-            nl_msg *msg = nlmsg_convert((nlmsghdr *)fpm_msg_data(hdr));
+            bool isRaw = false;
+
+            nlmsghdr *nl_hdr = (nlmsghdr *)fpm_msg_data(hdr);
+
+            /*
+             * EVPN Type5 Add Routes need to be process in Raw mode as they contain 
+             * RMAC, VLAN and L3VNI information.
+             * Where as all other route will be using rtnl api to extract information 
+             * from the netlink msg.
+           * */
+            isRaw = isRawProcessing(nl_hdr);
+
+            nl_msg *msg = nlmsg_convert(nl_hdr);
             if (msg == NULL)
+            {
                 throw system_error(make_error_code(errc::bad_message), "Unable to convert nlmsg");
+            }
 
             nlmsg_set_proto(msg, NETLINK_ROUTE);
-            NetDispatcher::getInstance().onNetlinkMessage(msg);
+
+            if (isRaw)
+            {
+                /* EVPN Type5 Add route processing */
+                NetDispatcher::getInstance().onNetlinkMessageRaw(msg);
```

Comment:
>NetDispatcher::getInstance().onNetlinkMessageRaw(msg); [](start = 16, length = 54)

Could you call onMsgRaw directly? So there is no need to change sonic-swss-common